fibers.texi

```diff
@@ -532,8 +532,8 @@ The new fiber will run concurrently with other fibers.
 The fiber will be added to the current scheduler, which is usually
 what you want.  It's also possible to spawn the fiber on a specific
 scheduler, which is useful to ensure that the fiber runs on a
-different kernel thread.  In that case, pass the @code{#:scheduler}
-keyword argument.
+different kernel thread.  In that case, pass the optional
+@code{scheduler} argument.
```

Comment:
hi! i believe optional argument is actually called `sched` instead of `scheduler`.

Reply:
Looking at the documentation I believe it actually makes sense to change the optional argument to be `scheduler`.

Reply:
hi, yes, you are right!
I have updated the pull request and added another commit which renames `sched` to `scheduler`.
I hope thats ok.